### ISSUE TITLE
False positive in definition after update to typescript-eslint v5

### ISSUE THREAD

**Issue by alumni** [gund/eslint-plugin-deprecation#36]
After updating typescript-eslint 4.33.0 to 5.0.0, all deprecated class members are show an error in the definition.

Example:
```ts
class Dog {
  sprint();

  /**
   * @deprecated Use {@link sprint} instead
   */
  walk();
}
```

The method `walk` would be highlighted as deprecated in the class definition.

**Comment by gund** [comment]
This was released in v1.3.0. Could you please check if it's fixed for you?
